src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -41,6 +43,7 @@ public function onKernelException(GetResponseForExceptionEvent $event)
     {
         $exception = $event->getException();
         $request = $event->getRequest();
+        $eventDispatcher = func_num_args() > 2 ? func_get_arg(2) : null;
```

Comment:
just define the proper arguments instead

Reply:
Can I define them as non-optional?

Reply:
@stof that'd be a BC break...

Reply:
hmm, our listener methods are indeed not marked as `@internal` 😢